apps/src/studio/skins.js

```diff
@@ -313,7 +331,96 @@ function loadHoc2015(skin, assetUrl) {
+     [0x00, 0x14, 0x00, 0x01, 0x01, 0x00, 0x05, 0x00],
+     [0x00, 0x02, 0x00, 0x01, 0x01, 0x00, 0x15, 0x00], 
+     [0x00, 0x03, 0x00, 0x00, 0x00, 0x00, 0x02, 0x00], 
+     [0x00, 0x24, 0x25, 0x01, 0x00, 0x34, 0x35, 0x00], 
+     [0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00]];
+
+  skin.maze3 = 
+    [[0, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00], 
+     [0, 0x01, 0x01, 0x01, 0x00, 0x01, 0x01, 0x00], 
+     [0, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00], 
+     [0, 0x01, 0x00, 0x01, 0x01, 0x01, 0x01, 0x00],
+     [0, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00], 
+     [0, 0x01, 0x01, 0x01, 0x01, 0x01, 0x00, 0x00], 
+     [0, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00], 
+     [0, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00]];
+
+  skin.maze4 = 
+    [[0,  0,  0,  0,  0,  0,  0,  0], 
+     [0, 01,  0, 01,  0, 01,  0,  0], 
+     [0, 01,  0, 01,  0, 01, 00,  0], 
+     [0, 01,  0, 01,  0, 01,  0,  0], 
+     [0, 01,  0, 01, 00, 01, 00,  0], 
+     [0, 01,  0, 01,  0, 01,  0,  0],  
+     [0, 01, 00, 01, 00, 01, 00,  0], 
+     [0, 01,  0,  0,  0, 01,  0,  0]];
+
+  skin.maze5 = 
+    [[0,  0,  0,  0,  0,  0,  0,  0], 
+     [0, 01,  0, 01,  0, 01,  0, 01], 
+     [0,  0,  0,  0,  0,  0,  0,  0], 
+     [0, 01,  0, 01,  0, 01,  0, 01], 
+     [0,  0,  0,  0,  0,  0,  0,  0],
+     [0, 01,  0, 01,  0, 01,  0, 01],  
+     [0,  0,  0,  0,  0,  0,  0,  0], 
+     [0, 01,  0, 01,  0, 01,  0, 01]];
+
+  skin.maze_blank = 
+    [[0,  0,  0,  0,  0,  0,  0,  0], 
+     [0,  0,  0,  0,  0,  0,  0,  0], 
+     [0,  0,  0,  0,  0,  0,  0,  0], 
+     [0,  0,  0,  0,  0,  0,  0,  0],  
+     [0,  0,  0,  0,  0,  0,  0,  0], 
+     [0,  0,  0,  0,  0,  0,  0,  0],   
+     [0,  0,  0,  0,  0,  0,  0,  0],  
+     [0,  0,  0,  0,  0,  0,  0,  0]];
+
+  skin.maze6 = 
+    [[0,  0,  0,  0,  0,  0,  0,  0], 
+     [0, 01, 01,  0,  0,  0, 01,  0], 
+     [0, 01, 01,  0,  0, 01, 01,  0], 
+     [0,  0,  0,  0,  0, 01, 01,  0],  
+     [0,  0, 01, 01,  0,  0,  0,  0], 
+     [0,  0, 01, 01,  0,  0,  0, 01],   
+     [0,  0, 01,  0,  0,  0, 01, 01],  
+     [0,  0,  0,  0,  0,  0, 01, 01]];
+
+  skin.maze7 = 
+    [[0x00, 0x30, 0x43, 0x43, 0x31, 0x00, 0x01, 0x02], 
+     [0x00, 0x42, 0x00, 0x00, 0x42, 0x00, 0x00, 0x03], 
+     [0x00, 0x42, 0x00, 0x00, 0x42, 0x00, 0x00, 0x04], 
+     [0x00, 0x42, 0x00, 0x00, 0x42, 0x00, 0x00, 0x14],  
+     [0x00, 0x32, 0x43, 0x40, 0x33, 0x00, 0x24, 0x25], 
+     [0x00, 0x00, 0x00, 0x42, 0x00, 0x00, 0x00, 0x10],   
+     [0x00, 0x00, 0x00, 0x32, 0x43, 0x00, 0x00, 0x11],  
+     [0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x20, 0x21]];
+
+  skin.maze8 = 
+    [[01,  0,  0, 01, 01,  0,  0, 01], 
+     [ 0,  0,  0,  0,  0,  0,  0,  0], 
+     [01, 01, 01,  0,  0, 01, 01, 01], 
+     [ 0,  0,  0,  0,  0,  0,  0,  0],  
+     [01, 01, 01,  0,  0, 01, 01, 01], 
+     [ 0,  0,  0,  0,  0,  0,  0,  0],   
+     [01, 01, 01,  0,  0, 01, 01, 01],  
+     [ 0,  0,  0,  0,  0,  0,  0,  0]];
+
+  skin.maze9 =
```

Comment:
Do you want to expose any of these new wall types in the dropdowns or tooltip strings?